amd/src/cv_launch.js

```diff
@@ -50,7 +50,7 @@ define(['jquery'], function($) {
 
                 var loading = '<div class="turnitinsim_Loading">' +
                         '<div class="turnitinsim_loadingLogo">' +
-                            '<img src="' + icon + '">' +
+                            '<img src="' + icon + '" width="350">' +
```

Comment:
px? Out of interest, is it way too big/small without a width attribute?

Reply:
@jmcgettrick I don't think you need to add px to the width attribute. It is massive without changing the width. I suppose I could resize the image itself. Not sure what's best.

Reply:
Na it's cool, just leave it!